examples/webgl_loader_3dm.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - loaders - Rhino 3DM loader</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+
+	<body>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - Rhino 3DM loader
+		</div>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+			import { CSS2DRenderer } from './jsm/renderers/CSS2DRenderer.js';
+
+			import { OrbitControls } from './jsm/controls/OrbitControls.js';
+			import { Rhino3dmLoader } from './jsm/loaders/3DMLoader.js';
+
+			import { GUI } from './jsm/libs/dat.gui.module.js';
+
+			var container, controls;
+			var camera, scene, renderer, labelRenderer;
+			var gui;
+
+			init();
+			animate();
+
+			function init() {
+
+				THREE.Object3D.DefaultUp = new THREE.Vector3( 0, 0, 1 );
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 1000 );
+				camera.position.set( 26, - 40, 5 );
+
+				scene = new THREE.Scene();
+
+				scene.add( new THREE.AmbientLight( { intensity: 0.0001 } ) );
+
+				var directionalLight = new THREE.DirectionalLight( 0xffffff );
+				directionalLight.position.set( 0, 0, 2 );
+				directionalLight.castShadow = true;
+				directionalLight.intensity = 2;
+				scene.add( directionalLight );
+
+				var loader = new Rhino3dmLoader( );
+				loader.setLibraryPath( 'jsm/libs/rhino3dm/' );
+				
+				loader.load( 'models/3dm/Rhino_Logo.3dm', function ( object ) {
+
+					console.log( object );
+					scene.add( object );
+					initGUI( object.userData.layers );
+
+                } );
```

Comment:
Minor formatting issue.

Reply:
I found a bunch of non empty lines...strange since I passed it through a linter and it was ok. Anyways, next commit hopefully solves that.